nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/organization/snippets/organization_form.html

```diff
@@ -0,0 +1,23 @@
+{%  ckan_extends %}
+
+{% block basic_fields %}
+<div class="hide-slug-preview">
```

Comment:
Missleading class name. One would assume that this class hides the div. 
Better to use something like "napote-org-form" or "napote-org-basic-fields" as a wrapper class name